openmetadata-ui/src/main/resources/ui/src/components/TableQueries/QueryCard.tsx

```diff
@@ -215,6 +232,13 @@ const QueryCard: FC<QueryCardProp> = ({
               onChange={handleQueryChange}
             />
           </div>
+          {isSqlInvalid && (
+            <Paragraph className="ml-8 p-l-md error-text">
+              {t('message.field-text-is-invalid', {
+                fieldText: t('label.sql-uppercase-query'),
+              })}
+            </Paragraph>
+          )}
```

Comment:
Can we use `Form.Item` to show validation issue instead of a custom para?

Reply:
done